apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -33,24 +29,36 @@ const CREATE_EVENT = gql`
   }
 `;
 
+const UPLOAD_PHOTO = gql`
+  mutation UploadFile($file: Upload!) {
+    uploadFile(file: $file) {
+      id
+      mediaType
+    }
+  }
+`;
+
 /**
  * This component renders the create event page
  * @param {History} history
  */
-const CreateEventPage = ({ history }) => {
+const CreateEventPage = ({ history }: CreateEventProps) => {
   // Reference Id for the toast
   const toastId = useRef(null);
   // GraphQL mutation hook to create events
   const [createEvent] = useMutation(CREATE_EVENT);
+  const [uploadFile] = useMutation(UPLOAD_PHOTO);
   // Image of Event
-  const [imageFile, setImageFile] = useState(null);
-
+  const [imageURLFile, setImageURLFile] = useState(null);
+  const [file, setFile] = useState<File>(null);
```

Comment:
If possible, add type annotation to hook to get that sweet autocomplete. I'm not sure what the mutation hooks need as type, so my suggestion might be wrong:
```ts
  const [createEvent] = useMutation<IEvent>(CREATE_EVENT);
  const [uploadFile] = useMutation<Upload>(UPLOAD_PHOTO);
  // Image of Event
  const [imageURLFile, setImageURLFile] = useState<string>(null);
  const [file, setFile] = useState<File>(null);
```

Reply:
That sweet sweet auto complete